openedx/features/wikimedia_features/messenger/templates/messenger.html

```diff
@@ -15,6 +15,7 @@
 </%block>
 
 <%block name="body">
+    <link rel="stylesheet" type="text/css" href="/static/css/ReactToastify.css" />
```

Comment:
`link` tag will be added in the `head_extra` block.

https://github.com/wikimedia/edx-platform/blob/3b35dbf7993d23eb3a7ed559c876d10068d85118/lms/templates/support/contact_us.html#L20